java_interop/protobuf-triple/java/java-server/run.sh

```diff
@@ -0,0 +1 @@
+mvn -e clean compile exec:java -Dexec.mainClass="org.apache.dubbo.sample.Provider"
```

Comment:
Better use `mvn wrapper`

Reply:
https://github.com/apache/dubbo-samples/tree/master/11-quickstart

check mvn wrapper configuration in this directory.